config/deploy.rb

```diff
@@ -5,7 +5,7 @@
 set :repo_url, "https://github.com/pulibrary/marc_liberation.git"
 
 # Default branch is :master
-ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+set :branch, ENV['BRANCH'] || 'master'
```

Comment:
I am 👍 on this change — and assume it works with pulbot because we do this in other projects.  There's nothing in the README about deployment now, but do you think it would be good to add something?

Reply:
@escowles yeah, I changed this (copied from figgy) because I couldn't get it to actually ask me. not sure what is up with that. I just pushed a note to the README.